rmgpy/solver/base.pyx

```diff
@@ -594,6 +596,7 @@ cdef class ReactionSystem(DASx):
         cdef int numCoreSpecies, numEdgeSpecies, numPdepNetworks, numCoreReactions
         cdef double stepTime, charRate, maxSpeciesRate, maxNetworkRate, maxEdgeReactionAccum, stdan
         cdef numpy.ndarray[numpy.float64_t, ndim=1] y0 #: Vector containing the number of moles of each species
+        cdef numpy.ndarray[numpy.float64_t, ndim=1] corerates,edgerates,leakrates,MW
```

Comment:
Please change the nomenclature. Suggestions: coreWeightRates, edgeWeightRates, MWNetworkLeakRates, MW